### DISCUSSION TITLE
[Community Task #2] Update WTF Cairo 1 tutorials

### DISCUSSION THREAD

**Discussion by AmazingAng** [WTFAcademy/WTF-Starknet#61] [General]
## Task: Update WTF Cairo 1 tutorial

Let's continue to create a minimalist Cairo 1 tutorial.

### Requirements:
- Have basic knowledge about Starknet.
- Have played with [starklings-cairo1](https://github.com/shramee/starklings-cairo1) or have written cairo1 contract in the past.
- Have some experience in rust
- Max 4 chapters / contributor
### Bounty
$60 per chapter

### Outline
0. How to set up a new Cairo project and deploy it [hasselalcala](https://github.com/hasselalcala)
1. Modules [hasselalcala](https://github.com/hasselalcala)
2. ABI [hasselalcala](https://github.com/hasselalcala)
3. Calling Other Contract [hasselalcala](https://github.com/hasselalcala)
4. Create contract [robertkodra](https://github.com/robertkodra)
5. Hash @AmazingAng 
6. Selector
7. System calls @enehizy 
8. Test [robertkodra](https://github.com/robertkodra)
9. Account [enehizy](https://github.com/enehizy)
10. ERC20 [robertkodra](https://github.com/robertkodra)
11. ERC721 [robertkodra](https://github.com/robertkodra)

### How to apply
Reply below with the chapters you want to complete and a brief intro of yourself.

### Resources
1. WTF Cairo  https://github.com/WTFAcademy/WTF-Cairo
2. WTF Solidity: https://github.com/AmazingAng/WTF-Solidity/blob/main/Languages/en/README.md
3. Starklings Cairo1: https://github.com/shramee/starklings-cairo1

**Comment by hasselalcala** [comment]
Hi! I am Hassel. 

I already finished all the starklings exercise, have experience with cairo1 contract and participate in two starknet basecamps, in the second one as a teacher. 

I want to take 0, 1, 2 and 3 chapters. (Please confirm if I can start to work in this)

Thank you!

**Reply by AmazingAng** [reply]
You can start! LFG!

**Comment by robertkodra** [comment]
Hello, 

I can help with 4/8/10/11. Not sure what's the max, but let me know. 

Thanks

**Reply by AmazingAng** [reply]
4 is great

**Comment by AmazingAng** [comment]
I will take 5.
@enehizy already finished 9.

**Reply by enehizy** [reply]
Thank you

**Comment by enehizy** [comment]
i would like to do the system calls chapter 7

**Reply by AmazingAng** [reply]
Great!

**Comment by gorbunova13** [comment]
thanks!

**Comment by codeWhizperer** [comment]
Hello, I would like to work on 6.
